Roles/Impostor/DollMaster.cs

```diff
@@ -264,6 +355,44 @@ public override bool OnCheckReportDeadBody(PlayerControl reporter, GameData.Play
         return false;
     }
 
+    // A fix when the DollMaster or Possessed Player DC's from the game.
+    // Untested!
+    public override void OnPlayerLeft(InnerNet.ClientData clientData)

```

Comment:
I forgot to remove this since it's no longer in use.
Now leaving a player from the game is the same as murder (But as suicide)

![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/a95ae3ae-8701-41c7-a6cc-b3c7158acbc8)

Reply:
So what is a good way to check if a player leaves the game?

Reply:
> So what is a good way to check if a player leaves the game?

`killer.PlayerId == target.PlayerId`

Reply:
I temporarily just commented out while I find a better way to execute it